shell/platform/tizen/tizen_native_window.cc

```diff
@@ -6,6 +6,49 @@
 
 #include "flutter/shell/platform/tizen/tizen_log.h"
 
+void LogLastEGLError() {
```

Comment:
Should we add a copyright notice at the top when copying code from elsewhere?

Reply:
yes